### PR TITLE
Use articles of association goal as introduction

### PR DESCRIPTION
**Pull request by bvhme**
This follows a discussion in the board meeting where we discussed whether this sentence shouldn't reflect more closely what we do now instead of a more open statement. I hope this wording helps us achieve our immediate priorities more and I think it would be a worthwhile improvement.

I've taken the wording from [our translation of the articles of association](https://about.publiccode.net/organization/articles-of-association.html).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 <div class="mp-section">
   <div>
-    <h1 class="title">We enable public-purpose software and policy that is open and collaborative.</h1>
+    <h1 class="title">We support collective development and maintenance of public code in order to reduce risk, increase quality, reduce cost and increase sustainability.</h1>
```
Comment (lscheske) [inline index.html:8]:
```suggestion
    <h1 class="title">We help public organizations collectively develop and maintain public code.</h1>
```

Comment (lscheske) [reply index.html:8]: Suggesting this even simpler change as the second part (benefits) are described in line 10.

**Review by lscheske — COMMENTED** [review]
* Suggested formulation that keeps the same meaning as words as from the articles of association, but made the sentence into an active one + added a subject.

File: index.html (hunk 1 above)
Comment (AlbaRoza) [reply index.html:8]: Very clear especially for a non native English speaker. Like it!!

Comment (nebster) [reply index.html:8]: I like this version, but I think part of the reason we changed it to begin with is that it does not explain what public code is, which is what "public-purpose software and policy that is open and collaborative" was meant to do.  Do we think we could, without using the language that seems to make people uncomfortable, define public code?

Comment (lscheske) [reply index.html:8]: @nebster that appears in a separate line below this "We define ‘public code’ as open source software developed by public organizations, together with the policy and guidance needed for reuse."

**Review by ElenaFdR — APPROVED** [review]
This looks like a great improvement in clarity!

**Pull request merged**